### ISSUE TITLE
Only top half of screen are mirrored / recorded - Poco F4-gt

### ISSUE THREAD

**Issue by pizi-0** [Genymobile/scrcpy#5582]
## Environment

 - **OS:** archlinux
 - **Scrcpy version: 3.0
 - **Installation method: aur
 - **Device model: poco f4-gt
 - **Android version: 14

Only top half of screen are mirrored / recorded for my poco f4-gt. This does not happens with my redmagic 8. I'm not sure what I did wrong.

When I launched 2 at the same time, the whole screen got mirrored.

```
 ~ scrcpy                                                 Mon 02 Dec 2024 05:50:48 PM +08
scrcpy 3.0 <https://github.com/Genymobile/scrcpy>
INFO: ADB device found:
INFO:     -->   (usb)  bf5351d                         device  21121210G
/usr/share/scrcpy/scrcpy-server: 1 file pushed, 0 skipped. 206.4 MB/s (90224 bytes in 0.000s)
[server] INFO: Device: [Xiaomi] POCO 21121210G (Android 14)
INFO: Renderer: opengl
INFO: OpenGL version: 4.6.0 NVIDIA 565.57.01
INFO: Trilinear filtering enabled
INFO: Texture: 1080x2400
```

https://github.com/user-attachments/assets/ad8c05ce-2a0f-46f8-9998-a37cbdfdbc6a

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by rom1v** [comment]
> recorded

Do you mean that if you record with `scrcpy --record=file.mp4`, the resulting file does not show the whole device?

**Comment by rom1v** [comment]
Other tests to determine the cause:

 - does <kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>r</kbd> (resetting the capture/encoding) "fixes" the problem?
 - does it also happen in "floating" mode? (#3430)
 - what is the behavior with scrcpy 2.7?

**Comment by pizi-0** [comment]
> > > does Alt+Shift+r (resetting the capture/encoding) "fixes" the problem?
> > 
> > 
> > No.
> 
> Do you have a log in the console when you press these keys?
> 
> > [server] INFO: Video capture reset
```
scrcpy 3.0 <https://github.com/Genymobile/scrcpy>
INFO: ADB device found:
INFO:     -->   (usb)  bf5351d                         device  21121210G
/usr/share/scrcpy/scrcpy-server: 1 file pushed, 0 skipped. 123.2 MB/s (90224 bytes in 0.001s)
[server] INFO: Device: [Xiaomi] POCO 21121210G (Android 14)
INFO: Renderer: opengl
INFO: OpenGL version: 4.6.0 NVIDIA 565.57.01
INFO: Trilinear filtering enabled
INFO: Texture: 1080x2400
[server] INFO: Video capture reset
[server] INFO: Video capture reset
```

**Comment by rom1v** [comment]
Can you reproduce with:

```
adb shell screenrecord /sdcard/file.mp4
adb pull /sdcard/file.mp4
```

?

**Comment by pizi-0** [comment]
> Can you reproduce with:
> 
> ```
> adb shell screenrecord /sdcard/file.mp4
> adb pull /sdcard/file.mp4
> ```
> 
> ?

Recorded full screen

**Comment by rom1v** [comment]
It looks like a device issue anyway (especially if it also happens with scrcpy 2.7).

You could try with another video encoder (see `--list-encoders`)?
And also try starting the scrcpy while your device is in landscape mode.

**Comment by pizi-0** [comment]
> It looks like a device issue anyway (especially if it also happens with scrcpy 2.7).
> 
> You could try with another video encoder (see `--list-encoders`)? And also try starting the scrcpy while your device is in landscape mode.

Yeah apparently device issue. Changing encoders doesn't help, so I factory reset. That fixed it. Don't know what caused the issue tho.
Thank you for helping.
